patches/server/0010-Adventure.patch

```diff
@@ -7,17 +7,32 @@ Co-authored-by: zml <zml@stellardrift.ca>
 Co-authored-by: Jake Potrebic <jake.m.potrebic@gmail.com>
 
 diff --git a/src/main/java/com/destroystokyo/paper/PaperConfig.java b/src/main/java/com/destroystokyo/paper/PaperConfig.java
-index 1d03a79e9010bc514b72a81ba0ad4a62aeff1bb7..429b74474ced04d8dd8f038b8590b8dfe178bf4d 100644
+index 1d03a79e9010bc514b72a81ba0ad4a62aeff1bb7..1674b38c4f628c095c55862627fae85013bac2c9 100644
 --- a/src/main/java/com/destroystokyo/paper/PaperConfig.java
 +++ b/src/main/java/com/destroystokyo/paper/PaperConfig.java
-@@ -217,4 +217,9 @@ public class PaperConfig {
+@@ -64,8 +64,8 @@ public class PaperConfig {
+         commands = new HashMap<String, Command>();
+         commands.put("paper", new PaperCommand("paper"));
+ 
+-        version = getInt("config-version", 20);
+-        set("config-version", 20);
++        version = getInt("config-version", 21);
```

Comment:
Do this in the config patch; dropping a patch does not mean rolling back a version.